### ISSUE TITLE
Can't match addons with Lua 5.1 (but with Lua 5.3.4)

### ISSUE THREAD

**Issue by jukx** [Phanx/wow-addon-updater#3]
Whenever I try to match an addon via update.lua using Lua 5.1, I get an error while with 5.3.4 it _seems_ (see other issue) to work.

```
lua5.1: update.lua:24: bad argument #3 to 'format' (string expected, got nil)
stack traceback:
	[C]: in function 'format'
	update.lua:24: in function 'formatMessage'
	update.lua:48: in function 'print'
	update.lua:85: in function <update.lua:84>
	(tail call): ?
	update.lua:181: in function 'scanAddons'
	update.lua:231: in function 'start'
	update.lua:240: in main chunk
	[C]: ?
```

**Comment by Phanx** [comment]
Possibly fixed in bd158dc. The error is caused by nil values getting passed to the message formatter; if nothing else, this change will at least let you see *what* the nil value is, which will point to the underlying problem.

**Comment by Phanx** [comment]
Going to assume this is fixed; let me know if it's not.
